src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -43,8 +46,8 @@ class ParticipantClinical extends React.Component {
       });
     }
 
-    let diagnoses = get(this.props.participant, 'diagnoses.hits.edges', []).map(ele =>
-      Object.assign({}, get(ele, 'node', {})) //copy obj
+    let diagnoses = get(this.props.participant, 'diagnoses.hits.edges', []).map(
+      ele => Object.assign({}, get(ele, 'node', {})), //copy obj
```

Comment:
Remove comma

Reply:
Auto-format by Prettify.

It makes the code uniform, but it sometimes makes a :hankey:

Reply:
good point